jekyll/_api/v1-reference.md

```diff
@@ -179,7 +179,7 @@ The branch name should be url-encoded.
 
 You can download an individual artifact file via the API with an API-token authenticated HTTP request.
 
-```sh
+```shell
```

Comment:
`sh` is not a supported language code for PrismJS. Have to change all of them to `shell`